frontend/app/services/TouchpointBackend.scala

```diff
@@ -78,7 +69,7 @@ object TouchpointBackend {
     val pids = Config.productIds(restBackendConfig.envName)
 
     val catalogRestClient: SimpleClient[Future] = new SimpleClient[Future](restBackendConfig, RequestRunners.configurableFutureRunner(60.seconds))
-    val newCatalogService = new subsv2.services.CatalogService[Future](pids, catalogRestClient, Await.result(_, 60.seconds), restBackendConfig.envName)
+    val newCatalogService = new subsv2.services.CatalogService[Future](pids, FetchCatalog.fromZuoraApi(catalogRestClient), Await.result(_, 60.seconds), restBackendConfig.envName)
```

Comment:
This should just preserve the existing behaviour, see: https://github.com/guardian/membership-common/pull/576